solutions/usaco-1088.mdx

```diff
@@ -11,3 +11,40 @@ author: Melody Yu (Video), Tyler Xiao (Video)
 
 Note: The video solution might not be the same as other solutions. Code in Java.
 <Youtube id="7eMHxpDQJEw" />
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(N^2)$
+
+<LanguageSection>
+
+<PySection>
+
+```py
+n = int(input())
+# grid_by_rows[i] is the ith row
+grid_by_rows = [list(map(int, input().split())) for _ in range(n)]
+# grid_by_cols[i] is the ith column
+grid_by_cols = zip(*grid_by_rows)
+
+rows_alternate, cols_alternate = 0, 0
+
+for i in range(n):
+	# each row has an alternating pattern of cow and no cow
+	row = grid_by_rows[i]  # the ith row
+	# row[::2] is a list of every other element, starting with index 0
+	# row[1::2] is a list of every other element, starting with index 1
```

Comment:
don't think these slicing comments are necessary tbh

Reply:
i'll leave the row ones in and delete the col ones since it's kind of repetitive